src/shared_realm.hpp

```diff
@@ -213,7 +213,17 @@ class Realm : public std::enable_shared_from_this<Realm> {
     // encryption key will raise an exception.
     static SharedRealm get_shared_realm(Config config);
 
+    // This is called by RealmCoordinator to update a Realm to a given schema, using
+    // the Realm's pre-set schema mode.
+    // By setting notify to false, schema changed function won't be called to ensure
+    // the notification will only be sent after Realm::get_shared_realm() returns.
+    void update_schema_notify(Schema schema, uint64_t version,
```

Comment:
It looks like this declaration wasn't removed when the implementation and calls to it were removed.